src/inspector/view/qml/MuseScore/Inspector/InspectorForm.qml

```diff
@@ -70,9 +71,20 @@ Rectangle {
     Flickable {
         id: flickableArea
 
-        anchors.fill: parent
+        anchors.top: root.top
+        anchors.right: root.right
+        anchors.left: root.left
         anchors.margins: 12
 
+        interactive: true
+        clip: true
+        flickableDirection: Flickable.VerticalFlick
+        boundsBehavior: Flickable.StopAtBounds
+        maximumFlickVelocity: 1000
+
+        height: root.height
```

Comment:
anchors.top/right/left + height equivalent to anchors.fill
So, please return anchors.fill: parent